components.css

```diff
@@ -750,3 +750,15 @@
     align-items: center;
     margin-left: auto;
 }
+
+.modal{
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    height: auto;
+    background-color: var(--white);
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    border-radius: 1rem;
```

Comment:
For defining sizes pls be consistent on 1 unit like if using rem then try to use rem throughout, although its not a rule but it will make the code look good.

Reply:
I have not used any other unit except rem.